src/auth/axios.ts

```diff
@@ -38,7 +46,7 @@ const responseErrorInterceptor = (error: AxiosError) => {
     isTokenValid(tokens.result.refreshToken) &&
     !(error.config as any)?._retry
   ) {
-    refresh(tokens.result.refreshToken)(
+    await refresh(tokens.result.refreshToken)(
```

Comment:
📝 If you don't wait, the original request will be returned and made again with the old access token, resulting in yet another unauthorized access error.